### PR TITLE
Moves the global preferences to the sidebar

### PR DESCRIPTION
**Pull request by cmgustavo**

### CODE REVIEW

File: public/views/preferencesGlobal.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+<div 
+  class="topbar-container" 
+  ng-include="'views/includes/topbar.html'"
+  ng-init="titleSection='Global Preferences'; closeToHome = true; noColor = true">
+</div>
+
+
+<div class="content preferences" ng-controller="preferencesGlobalController as prefGlobal" ng-init="prefGlobal.init()">
+
+  <ul ng-show="!index.noFocusedWallet" class="no-bullet m0 ">
```
Comment (matiu) [inline public/views/preferencesGlobal.html]: Maybe we should remove the `ng-show`?

Comment (cmgustavo) [inline public/views/preferencesGlobal.html]: Agree.

**Comment by cmgustavo** [comment]
Updated. Ready for review

**Comment by matiu** [comment]
reviewing.

**Comment by matiu** [comment]
Great work.

Review code and tested on browser.

**Pull request merged**